apps/financial-aid/web-osk/src/components/ServiceCenter/ServiceCenter.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { Text } from '@island.is/island-ui/core'
+
+import {
+  ContentContainer,
+  Footer,
+} from '@island.is/financial-aid-web/osk/src/components'
+
+import { ServiceCenter as IServiceCenter } from '@island.is/financial-aid/shared/data'
+import { useLogOut } from '@island.is/financial-aid-web/osk/src/utils/useLogOut'
+
+interface Props {
+  serviceCenter?: IServiceCenter
+}
+
+const ServiceCenter = ({ serviceCenter }: Props) => {
+  const logOut = useLogOut()
+
+  return (
+    <>
+      <ContentContainer>
+        <Text as="h1" variant="h2" marginBottom={[1, 1, 2]}>
+          Fjárhagsaðstoð Hafnarfjarðar
```

Comment:
Is this the right header?

Reply:
According to design, then yess, I will comment on this in figma 😅